compiler/src/iree/compiler/Codegen/Common/ForOpCanonicalizationPass.cpp

```diff
@@ -198,7 +198,7 @@ struct PackForOpInductionVarVector final : public OpRewritePattern<scf::ForOp> {
         continue;
       }
       int64_t numElements = ShapedType::getNumElements(iterType.getShape());
-      int64_t bitWidth = iterType.getElementType().getIntOrFloatBitWidth();
+      int64_t bitWidth = IREE::Util::getTypeBitWidth(iterType.getElementType());
       int64_t totalBits = numElements * bitWidth;
```

Comment:
there's a utility for total storage size that does this rounding too, so any place we are doing element count * bit width should use that instead

Reply:
getRoundedPhysicalStorageSize

Reply:
Oh nice, I didn't know this existed. There are some other places we should update to use this as well.

Reply:
Ok I thought about it a bit more (and should have checked the lit tests first) but I don't think the notion of a physical storage size applies to a vector in this context. We probably need to revisit packing/padding of non-byte aligned types to make sure they are consistent between codegen and what the host side does.